Resources/MixedModels/qml/MixedModelsLMM.qml

```diff
@@ -535,13 +535,13 @@ Form {
 			label:	qsTr("P-value adjustment")
 			values:
 			[
-				{ label: "Holm",				value: "holm"},
-				{ label: "Multivariate-t",		value: "mvt"},
-				{ label: "Scheffe",				value: "scheffe"},
-				{ label: "Tukey",				value: "tukey"},
-				{ label: "None",				value: "none"},
-				{ label: "Bonferroni",			value: "bonferroni"},
-				{ label: "Hommel",				value: "hommel"}
+				{ label: "Holm",					value: "holm"},
```

Comment:
Wouldn't this also require `qsTr()` so it can be adapted in other writing systems?